src/main/java/org/broadinstitute/consent/http/db/DatasetDAO.java

```diff
@@ -43,25 +44,28 @@ public interface DatasetDAO extends Transactional<DatasetDAO> {
     String CHAIRPERSON = Resource.CHAIRPERSON;
 
     @SqlUpdate(
-            "INSERT INTO dataset "
-            + "(name, create_date, create_user_id, update_date, "
-                + "update_user_id, object_id, active, dac_id, alias, data_use) "
-            + "(SELECT :name, :createDate, :createUserId, :createDate, "
-                + ":createUserId, :objectId, :active, :dacId, COALESCE(MAX(alias),0)+1, :dataUse FROM dataset)")
+            """
+            INSERT INTO dataset
+                (name, create_date, create_user_id, update_date,
+                update_user_id, object_id, open_access, active, dac_id, alias, data_use)
+            (SELECT :name, :createDate, :createUserId, :createDate,
+                :createUserId, :objectId, :openAccess, :active, :dacId, COALESCE(MAX(alias),0)+1, :dataUse FROM dataset)
+            """)
     @GetGeneratedKeys
     Integer insertDataset(
         @Bind("name") String name,
         @Bind("createDate") Timestamp createDate,
         @Bind("createUserId") Integer createUserId,
         @Bind("objectId") String objectId,
+        @Bind("openAccess") Boolean openAccess,
```

Comment:
Not sure if it would reduce the PR size, but it could be a property instead of a top level field on dataset. The only benefit there would be to trim the sql/dao changes down a bit

Reply:
Hmm, it's up to you, I could see it either way. It feels like being a top-level property makes sense, because it's fundamental to the operation of the dataset. But I could see it being a property as well, because it's a lot simpler.

Reply:
I'm nervous about adding something this important as a last-minute addition without going through all the uses of dataset to ensure it doesn't have any side effects. If it were a property, then the risk is much lower. I don't see it as a primary aspect of a dataset - it seems to have the same level of importance as many of the other properties that we're capturing separately as props.

Reply:
Sure, can do. I'll make it a property.